spec/tree_spec.rb

```diff
@@ -1,14 +1,53 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
-  it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
-  end
-end
+describe 'AppleTree' do
+	let(:tree) { AppleTree.new }	
```

Comment:
Adjust your editor. At CMM we wont approve your code review if its anything other than 2 spaces.